.pre-commit-config.yaml

```diff
@@ -9,27 +9,12 @@ repos:
     hooks:
       - id: black
         language_version: python3.7
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.2.1
+  - repo: local
     hooks:
     - id: prettier
-      args: [--write]
-      types_or: [javascript, jsx, ts, tsx]
+      name: prettier
+      language: system
+      types_or: [javascript,jsx,ts,tsx,markdown,html,scss,css,yaml]
       files: ^src/ts/src
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v7.11.0
-    hooks:
-    - id: eslint
-      args: [--fix, --config src/ts/.eslintrc.js. --ignore-path src/ts/.eslintignore, --max-warnings 45]
-      additional_dependencies:
-      - eslint@7.11.0
-      - eslint-plugin-react@7.22.0
-      - eslint-plugin-react-hooks@4.2.0
-      - eslint-config-prettier@7.1.0
-      - babel-eslint@10.1.0
-      - "@typescript-eslint/eslint-plugin@4.5.0"
-      - "@typescript-eslint/parser@4.5.0"
-      types_or: [javascript, jsx, ts, tsx]
-      files: ^src/ts/src
-
+      entry: npx prettier --write
```

Comment:
runs `npx prettier --write` directly, instead of installing via `https://github.com/pre-commit/mirrors-prettier`